common/unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -746,10 +751,6 @@ public void trim() {
 
   @Test
   public void trimLeft() {
-    assertEquals(fromString("  hello "), fromString("  hello ").trimLeft(fromString("")));
-    assertEquals(fromString(""), fromString("a").trimLeft(fromString("a")));
-    assertEquals(fromString("b"), fromString("b").trimLeft(fromString("a")));
-    assertEquals(fromString("b"), fromString("b").trimLeft(fromString("a")));
```

Comment:
I thought those test cases are too simple. I will put it back and remove the duplicate one. Thanks.